experiments/integrated/fluxnet/plot_utils.jl

```diff
@@ -0,0 +1,77 @@
+"""Plotting utilities for the integrated fluxnet site experiments"""
+
+using Interpolations
+using Plots
+using StatsBase
+
+S_PER_DAY = 86400 # Number of seconds in a day
+
+"""This function uses interpolation to convert a time series of data at 
+any regular time interval to half hourly data. It takes in the data, the 
+vector giving the time stamps of the data in seconds, and the number of days 
+in the data series. It returns a vector of the data interpolated to half
+hourly intervals."""
+function interp_hh(data::Vector, timeseries::Vector, num_days::Int64)
+    # Rescale the data to a half hourly interval by linear interpolation
+    interp = LinearInterpolation(timeseries, data)
+    half_hourly = 0.5*3600:0.5*3600:24 * num_days * 3600
+    hh_data = [interp[i] for i in half_hourly]
+    return hh_data
+end
+
+"""This function takes in a vector of half-hourly data and the number of days 
+in the data series and returns the vector giving the diurnal average of 
+the data over the time period. (For data which is not half-hourly, use 
+interp_hh first.)"""
+function diurnal_avg(hh_series::Vector, num_days::Int64)
```

Comment:
These names are good, but it could be helpful to call this "hh_to_diurnal_avg"? and the above to "interp_to_hh", and below:" compute_diurnal_avg"